source/geh_common/src/geh_common/data_products/electricity_market_measurements_input/missing_measurements_log_metering_point_periods_v1.py

```diff
@@ -17,10 +17,14 @@
         # Metering point resolution: PT1H/PT15M
         T.StructField("resolution", T.StringType(), not nullable),
         #
-        # UTC time
+        # First time the metering point is connected or disconnected. UTC time
         T.StructField("period_from_date", T.TimestampType(), not nullable),
```

Comment:
Should we use "active_from_date" and "active_to_date"?

Reply:
I don't think that is an official term?

Reply:
I would consult Khatozen.

Reply:
@khatozen What is your comment here?